### PR TITLE
Create v1.10.2.md

### PR DESCRIPTION
**Pull request by yaron2**
Add 1.10.2 release notes.

### CODE REVIEW

**Review by mukundansundar — APPROVED** [review]
lgtm

File: docs/release_notes/v1.10.2.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+## Fix removing extra commits from master branch in 1.10.1
+
+### Problem
+
+This release does not address any known issue. The Dapr 1.10.1 hotfix contained additional commits from the `master` branch that were not part of the 1.10 release branch.
+The 1.10.2 hotfix guarantees that all changes in the release are tested and validated.
+
+### Impact
+
+There is no known impact to users.
```
Comment (ItalyPaleAle) [inline docs/release_notes/v1.10.2.md:12]: Shouldn’t we say here that there is an impact as the build contained code that hadn’t been thoroughly tested yet?

File: docs/release_notes/v1.10.2.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+### Root cause
+
+The v1.10.1 tag pulled in additional commits from the `master` branch, possibly due to being based off of `master` branch.
```
Comment (ItalyPaleAle) [inline docs/release_notes/v1.10.2.md:16]:
```suggestion
The v1.10.1 tag pulled in additional code changes from the `master` branch, that were not scheduled to be included in a release yet.
```

File: docs/release_notes/v1.10.2.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+### Solution
+
+Tagging a minor version release off of the `release-1.10` branch.
```
Comment (ItalyPaleAle) [inline docs/release_notes/v1.10.2.md:20]:
```suggestion
We are releasing Dapr v1.10.2 with the correct set of changes from the Dapr 1.10 branch, and deleting all artifacts for Dapr v1.10.1.
```

**Comment by yaron2** [comment]
Do not merge, @artursouza has thoughts around patching 1.10.1.

**Comment by mukundansundar** [comment]
Should this be merged and 1.10.2 cut again?

**Comment by artursouza** [comment]
I will still do 1.10.2 because @mukundansundar pointed out that users might not pick up the fixed 1.10.1 release if they don't set their PodTemplates to always pull image.

**Review by artursouza — APPROVED** [review]

**Pull request merged**